tsconfig.base.json

```diff
@@ -9,16 +9,16 @@
     "target": "ESNext",
     "esModuleInterop": true,
     "moduleResolution": "node",
-    "baseUrl": ".",
     "types": ["react", "jest", "node", "trusted-types", "jest-extended"],
     "paths": {
-      "development-sandbox": ["test/lib/development-sandbox"],
-      "next-test-utils": ["test/lib/next-test-utils"],
-      "amp-test-utils": ["test/lib/amp-test-utils"],
-      "next-webdriver": ["test/lib/next-webdriver"],
-      "e2e-utils": ["test/lib/e2e-utils"],
-      "test-data-service/*": ["test/lib/test-data-service/*"],
-      "test-log": ["test/lib/test-log"]
+      "development-sandbox": ["./test/lib/development-sandbox"],
+      "next-test-utils": ["./test/lib/next-test-utils"],
+      "amp-test-utils": ["./test/lib/amp-test-utils"],
+      "next-webdriver": ["./test/lib/next-webdriver"],
+      "e2e-utils": ["./test/lib/e2e-utils"],
+      "test-data-service/*": ["./test/lib/test-data-service/*"],
+      "test-log": ["./test/lib/test-log"],
+      "test/*": ["./test/*"]
```

Comment:
This path is new and mostly what `baseUrl: "."` was doing since we have a bunch of `import { NextInstance } from 'test/lib/next-modes/base'`. Explicit paths are more helpful anyway since it makes dependency tracking explicit. Otherwise some random test could start importing `docs/some/internal/module`.

Might even be better to remove that catch-all in favor of moving the type to test utils and `import { NextInstance } from 'next-test-utils'`?

Reply:
> Might even be better to remove that catch-all in favor of import { NextInstance } from 'e2e-utils'?

Did just that.